test/functional/feature_taproot.py

```diff
@@ -1436,17 +1437,27 @@ def test_spenders(self, node, spenders, input_counts):
         self.log.info("  - Done")
 
     def run_test(self):
-        self.connect_nodes(0, 1)
-
         # Post-taproot activation tests go first (pre-taproot tests' blocks are invalid post-taproot).
         self.log.info("Post-activation tests...")
         self.nodes[1].generate(101)
         self.test_spenders(self.nodes[1], spenders_taproot_active(), input_counts=[1, 2, 2, 2, 2, 3])
 
-        # Transfer % of funds to pre-taproot node.
+        # Transfer funds to pre-taproot node.
         addr = self.nodes[0].getnewaddress()
-        self.nodes[1].sendtoaddress(address=addr, amount=int(self.nodes[1].getbalance() * 70000000) / 100000000)
-        self.nodes[1].generate(1)
+        rawtx = self.nodes[1].createrawtransaction(
+            inputs=[{
+                'txid': i['txid'],
+                'vout': i['vout']
+            } for i in self.nodes[1].listunspent()],
+            outputs={addr: self.nodes[1].getbalance()},
+        )
+        rawtx = self.nodes[1].signrawtransactionwithwallet(rawtx)['hex']
+        # Transaction is too large to fit into the mempool, so put it into a block
```

Comment:
Where does the error get thrown if you were to send the transaction normally? And why is it too large atm? Everything in the PR looks good, but I thought I should understand this before ack, and haven't been able to figure out

Reply:
You can find where it is thrown by running `git grep 'Transaction too large'`

Reply:
The transaction is too large because it has too many inputs